pkg/domain/domain.go

```diff
@@ -146,7 +146,7 @@ type Domain struct {
 	store           kv.Storage
 	infoCache       *infoschema.InfoCache
 	privHandle      *privileges.Handle
-	bindHandle      bindinfo.GlobalBindingHandle
+	bindHandle      atomic.Value
```

Comment:
```suggestion
	bindHandle      atomic.Pointer[bindinfo.GlobalBindingHandle]
```

Reply:
This avoids type conversion

Reply:
This is an interface, not a structure, so we can't use "Pointer" here.